python/tests/integration/test_methods.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from typing import Dict, List, Union
+
+from looker.sdk import methods as mtds
+from looker.sdk import models as ml
+
+
+@pytest.mark.skip(
+    reason="Currently fails because update_user assigns Nones to arguments that are not passed through."
+)
+def test_crud_user(client: mtds.LookerSDK):
+    """Test creating, retrieving, updating and deleting a user.
+    """
+
+    # Create user
+    user = client.create_user(
+        ml.WriteUser(first_name="John", last_name="Doe", is_disabled=True, locale="fr")
+    )
+    assert isinstance(user, ml.User)
+    assert isinstance(user.id, int)
+    assert user.first_name == "John"
+    assert user.last_name == "Doe"
+    assert user.is_disabled
+    assert user.locale == "fr"
+
+    # Update user
+    user_id = user.id
+    client.update_user(user_id, ml.WriteUser(is_disabled=False, locale="uk"))
+    client.create_user_credentials_email(
+        user_id, ml.WriteCredentialsEmail(email="john.doe@looker.com")
+    )
+
+    # Retrieve user details
+    user = client.user(user_id)
+    assert user.locale == "uk"
+    assert not user.is_disabled
+    assert isinstance(user.credentials_email, ml.CredentialsEmail)
+    assert user.credentials_email.email == "john.doe@looker.com"
+    assert user.first_name == "John"
+    assert user.last_name == "Doe"
+
+    # Delete user
+    resp = client.delete_user(user_id)
+    assert resp == ""
+
+
+def test_me_returns_correct_result(client: mtds.LookerSDK):
+    """me() should return the right user
+    """
+    me = client.me()
+    assert isinstance(me, ml.User)
+    assert isinstance(me.credentials_api3, list)
+    assert isinstance(me.credentials_api3[0], ml.CredentialsApi3)
+    assert len(me.credentials_api3) > 0
```

Comment:
I would swap L53/54 since a test failure where `me.credentials_api3` is empty will currently fail with an `IndexError`

Reply:
Good catch